rstudio-wsl/draft.Rmd

```diff
@@ -0,0 +1,157 @@
+
+> Installing, this may take a few minutes...                                                                                                             
+> 
+> Please create a default UNIX user account. The username does not need to match your Windows username.                                                  
+> For more information visit: `https://aka.ms/wslusers` 
+> 
+> Enter new UNIX username:                                                                                                                               
+  
+Provide a user name followed by password that will be used for both your WSL distribution and RStudio Server authentication later on.
+
+You can now close the WSL instance by typing the following command:
+   
+```{bash}
+exit
+```
+
+* **Note:** Unlike the PowerShell, you cannot use `Ctrl + V` shortcut key to paste the clipboard content into the WSL terminal. Alternatively, press mouse right-click button to emulate the paste function.
+
+## RStudio Server Setup {#rstudio-server-setup}
+
+Now that you have successfully installed a Linux distribution on your system via WSL, you can setup and run RStudio Server by following the procedures below:
+
+1. Type `wsl` in either PowerShell or CMD terminal to launch your installed linux distro.
+1. Make sure your Linux packages up-to-date using the following command (we will assume that you have installed a Ubuntu distribution):
+
+    ```{bash}
+    sudo apt-get update
+    sudo apt-get upgrade -y
+    ```
+
+    * **Note:** Provide the password of your user account that you created for the Linux system, not the Windows password to log in for your PC.
+
+1. Add the following apt credential to your repository list  for Getting the Latest R (v4.0.0 as of this writing):
+
+    ```{bash}
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+    sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/'
+    ```
+
+    * **Note:** More detailed information on installing the latest R for Linux can be found at [the official CRAN page](https://cran.r-project.org/bin/linux/ubuntu/){target="_blank"}.
+
+1. Install R with some essential dependencies for the RStudio Server and devtools:
+
+    ```{bash}
+    sudo apt install -y r-base r-base-core r-recommended r-base-dev gdebi-core build-essential libcurl4-gnutls-dev libxml2-dev libssl-dev
+    ```
+
+1. Obtain the download URL of the RStudio Server you want to install.
+
+    * **Note:** You can explore some RStudio download pages, such as [RStudio Release](https://rstudio.com/products/rstudio/download/#download){target="_blank"}, [RStudio Preview (close to release)](https://rstudio.com/products/rstudio/download/preview/){target="_blank"}, and [RStudio Daily](https://dailies.rstudio.com/){target="_blank"}. Make sure you choose RStudio *Server* (not Desktop) and match with your installed Linux distribution type and version.
```

Comment:
Let's only discuss installing the current stable build. Installing preview builds and daily builds is something people shouldn't do unless they have a specific reason. So give the download and install commands first (see below for the correct one to always get current stable release), then mention the RStudio Release download page where they can go find additional information if desired.